src/adm/settings.rs

```diff
@@ -123,16 +155,56 @@ impl From<&Settings> for AdmSettings {
 /// }
 /// ```
 ///
-impl From<&Settings> for HandlerResult<AdmFilter> {
-    fn from(settings: &Settings) -> Self {
+impl From<&mut Settings> for HandlerResult<AdmFilter> {
+    fn from(settings: &mut Settings) -> Self {
         let mut filter_map: HashMap<String, AdmAdvertiserFilterSettings> = HashMap::new();
+        let ignore_list = settings
+            .adm_ignore_advertisers
+            .clone()
+            .unwrap_or_else(|| "[]".to_owned());
         for (adv, setting) in AdmSettings::from(settings) {
             trace!("Processing records for {:?}", &adv);
             // map the settings to the URL we're going to be checking
             filter_map.insert(adv.to_lowercase(), setting);
         }
+        let ignore_list: HashSet<String> = serde_json::from_str(&ignore_list).map_err(|e| {
```

Comment:
We forgot one last thing: these need to be lowercase'd. Which suggests it could use a quick test too if it's not too much trouble.

Reply:
Not sure how to best test this, since the ignore list is basically doing the same thing as an invalid, just not reporting things out to Sentry. v. 0.19 doesn't include the inspection hacks. I'll add a test that verifies that the ignore list is lowercased for now.